components/builder-api/src/http/mod.rs

```diff
@@ -51,12 +51,6 @@ pub fn router(config: Arc<Config>) -> Result<Chain> {
         user_invitations: get "/user/invitations" => {
             XHandler::new(list_account_invitations).before(basic.clone())
         },
-        edit_user_invitation: put "/user/invitations/:invitation_id" => {
-            XHandler::new(accept_invitation).before(basic.clone())
-        },
-        delete_user_invitation: delete "/user/invitations/:invitation_id" => {
```

Comment:
It was never implemented in the UI, and this route is the wrong one - it needs to be on `origin`, like the accept flow is now. We're ready for it to work, but I didn't go the extra mile and implement it.